buildroot-external/package/generic_raw_uart/generic_raw_uart.hash

```diff
@@ -1,3 +1,3 @@
 # Locally computed
 sha256  b40930bbcf80744c86c46a12bc9da056641d722716c378f5659b9e555ef833e1  LICENSE
-sha256  f1741dfe9ea6ea37213c35c735aa899f11e4c2961bccd6d8198d4051b3c42c5d  generic_raw_uart-f691101c32435921a70c96075233b6d5e5eeab7d.tar.gz
+sha256  f8326439b6cc917d4c8ccc6fa670eb5a0ca50d43e08ca02140d9314bbc2429ac  generic_raw_uart-26b6aa161d69bd8dc6900f144b74b4ac884e9dab.tar.gz
```

Comment:
No need to bump generic_raw_uart to commit id `26b6aa1` as `[f691101](https://github.com/alexreinert/piVCCU/commit/f691101c32435921a70c96075233b6d5e5eeab7d)` is the last changed commit id that is relevant for the generic_raw_uart package. So please keep it at `f691101` because that's how we (in fact I) ensure/maintain the correct generic_raw_uart sources are used.